incubator/faucet/client/cli/tx.go

```diff
@@ -30,134 +21,61 @@ func GetTxCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	faucetTxCmd.AddCommand(flags.PostCommands(
-		GetCmdMint(cdc),
-		GetCmdMintFor(cdc),
-		GetCmdInitial(cdc),
-		GetPublishKey(cdc),
-	)...)
+	faucetTxCmd.AddCommand(
+		txMint(),
+		txMintFor(),
+	)
 
 	return faucetTxCmd
 }
 
-// GetCmdWithdraw is the CLI command for mining coin
-func GetCmdMint(cdc *codec.Codec) *cobra.Command {
-	return &cobra.Command{
+// GetCmdMint is the CLI command for mining coin
```

Comment:
nit: comment has typo should be 

`// GetCmdMint is the CLI command for minting coin`